ui-tests/cypress/integration/content/attributes-list-nested.spec.js

```diff
@@ -919,17 +965,19 @@ describe([Tag.GTS], 'Nested a in List Attribute', () => {
 
     it('Try to set standard validation (required) and check it in content validation', () => {
       addAttributeToContentType(attribute, {}, {mandatory: true});
-      navigateToContentForm();
-      currentPage.getContent()
-                 .fillBeginContent(CONTENT_WITH_LIST.description)
-                 .getSaveApproveAction().invoke('hasClass', 'disabled').should('be.true');
-      currentPage.getContent()
-                 .fillAttributes([{
-                   type: 'List',
-                   nestedType: attribute,
-                   value: [null]
-                 }])
-                 .getSaveApproveAction().invoke('hasClass', 'disabled').should('be.false');
+      navigateToContentForm()
+        .then(page => {
+          page.getContent().fillBeginContent(CONTENT_WITH_LIST.description);
+          page.getContent().getSaveApproveAction().click();
+          page.getContent().getAlertMessage().should('exist').and('contain.text', 'Mandatory');
+          page.getContent().fillAttributes([{
+            type: 'List',
+            nestedType: attribute,
+            value: [null]
+          }]);
+          page.getContent().submitApproveForm();
+          checkAndSavePublishedContentId();
+        });
     });
   });
 });
```

Comment:
Can you try to split each interaction in a separate `then` when handling the *page object*?